src/backends/github/API.js

```diff
@@ -85,8 +86,16 @@ export default class API {
     });
   }
 
+  truncateSlugForBranchName(slug) {
+    const slugMaxLength = this.branchNameMaxLength - CMS_BRANCH_PREFIX.length;
+    return slug.length > slugMaxLength
+      ? slug.substring(0, slugMaxLength)
```

Comment:
Can we just take out the ternary here? If the string is smaller than it, shouldn't the string just be left as-is? I'm not experienced enough with JS string behavior to know whether that is the case or not, but it seems to be.